activerecord/lib/active_record/transactions.rb

```diff
@@ -99,6 +99,16 @@ module Transactions
     # be propagated (after triggering the ROLLBACK), so you should be ready to
     # catch those in your application code.
     #
+    # Exceptions that trigger the ROLLBACK have a #message method that contains
```

Comment:
I wonder if we can rewrite this sentence to reflect that *all* Exceptions have a #message method.